lib/set/setJSONGraphs.js

```diff
@@ -118,18 +118,23 @@ function setJSONGraphPathSet(
 }
 /* eslint-enable */
 
+var _result = new Array(4);
 function setReference(
     root, node, messageRoot, message, requestedPath, optimizedPath,
     version, expired, lru, comparator, errorSelector, replacedPaths) {
 
     var reference = node.value;
-    optimizedPath.splice(0, optimizedPath.length);
```

Comment:
`splice` returns an array. Our engine isn't smart enough to optimize it out based on whether you're using it. `length = 0` is better for clearing the array.